src/blocks/CallToAction/index.tsx

```diff
@@ -24,32 +26,46 @@ export const CallToAction: React.FC<CallToActionProps> = props => {
   return (
     <BlockSpacing>
       <Gutter className={classes.callToAction}>
-        <div data-theme="dark">
-          <div className={classes.bgWrapper}>
-            <Gutter disableMobile className={classes.bgGutter}>
-              <div className={classes.bg1}>
-                {/* <div className={classes.pixelBG}>
-              <PixelBackground />
-            </div> */}
-              </div>
-            </Gutter>
-          </div>
-          <div className={[classes.contentWrap, 'grid'].filter(Boolean).join(' ')}>
-            <div className={'cols-8 cols-m-8'}>
+        <div className={[classes.wrapper].filter(Boolean).join(' ')}>
+          <BackgroundGrid ignoreGutter />
+          <div className={[classes.container, 'grid'].filter(Boolean).join(' ')}>
+            <div className={[classes.contentWrapper, 'cols-7 cols-m-8'].filter(Boolean).join(' ')}>
               <RichText content={richText} className={classes.content} />
             </div>
-            <div className={'cols-7 start-11 start-l-10 cols-m-8 start-m-1'}>
+            <div
+              className={[classes.linksContainer, 'cols-8 start-9 cols-m-8 start-m-1 grid']
+                .filter(Boolean)
+                .join(' ')}
+            >
+              <BackgroundScanline
+                className={[classes.scanline, 'cols-16 start-4 start-l-3 cols-m-8 start-m-1']
```

Comment:
I think I would do `cols-16 start-5 cols-m-8 start-m-1` instead for the `BackgroundScanline`.

Designs shows it starting at exactly the half way point between the two grid lines.